app/controllers/admin/users_controller.rb

```diff
@@ -48,8 +56,8 @@ def find_user
     @user = User.find(params[:id])
   end
 
-  def post_params
-    params.require(:user).permit(:first_name, :last_name, :email, :password)
+  def user_params
+    params.require(:user).permit(:first_name, :last_name, :phone, :country, :email, :password, :avatar)
```

Comment:
write this line like this

 params.require(:user).permit(
  :first_name, 
  :last_name, 
  :phone,
  :country, 
  :email,
  :password, 
  :avatar
)